server/command.go

```
@@ -174,6 +184,69 @@ func (p *Plugin) executeCommandQueue(args *model.CommandArgs) *model.CommandResp
 	return &model.CommandResponse{}
 }
 
+func calculateQueItemNumberAndUpdateOldItems(meeting *Meeting, args *model.CommandArgs, p *Plugin, hashtag string) (int, *model.CommandResponse) {
+	searchResults, appErr := p.API.SearchPostsInTeamForUser(args.TeamId, args.UserId, model.SearchParameter{Terms: &hashtag})
+	if appErr != nil {
+		return 0, responsef("Error calculating list number")
```

We should return an error here and `errors.Wrap` `appErr`